src/System.ComponentModel.Primitives/ref/System.ComponentModel.Primitives.cs

```diff
@@ -113,6 +117,7 @@ public enum DesignerSerializationVisibility
         Hidden = 0,
         Visible = 1,
     }
+    [System.AttributeUsageAttribute(System.AttributeTargets.Property | System.AttributeTargets.Method | System.AttributeTargets.Field | System.AttributeTargets.Event)]
```

Comment:
Did you use use a local copy of your tool to generate these?

Reply:
For these ones no, I copied from source.  The WPF/WinForms changes used the updated GenAPI.